en/docs/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.yaml

```diff
@@ -3672,308 +3729,387 @@ paths:
+                $ref: '#/components/schemas/APIExternalStoreList'
         404:
           $ref: '#/components/responses/NotFound'
-        406:
-          $ref: '#/components/responses/NotAcceptable'
+        500:
+          $ref: '#/components/responses/InternalServerError'
       security:
         - OAuth2Security:
             - apim:api_view
+            - apim:api_manage
       x-code-samples:
         - lang: Curl
           source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
-          "https://127.0.0.1:9443/api/am/publisher/v2/apis/96077508-fd01-4fae-bc64-5de0e2baf43c/wsdl"'
+          "https://127.0.0.1:9443/api/am/publisher/v3/apis/96077508-fd01-4fae-bc64-5de0e2baf43c/external-stores"'
 
-    put:
+  /apis/{apiId}/publish-to-external-stores:
+    post:
       tags:
-        - APIs
-      summary: Update WSDL Definition
+        - External Stores
+      summary: Publish an API to External Stores
       description: |
-        This operation can be used to update the WSDL definition of an existing API. WSDL to be updated can be passed as either "url" or "file".
-        Only one of "url" or "file" can be used at the same time. "file" can be specified as a single WSDL file or as a zip file which has a WSDL
-        and its dependencies (eg: XSDs)
-      operationId: updateWSDLOfAPI
+        This operation can be used to publish an API to a list of external stores.
+      operationId: publishAPIToExternalStores
       parameters:
         - $ref: '#/components/parameters/apiId'
+        - name: externalStoreIds
+          in: query
+          description: External Store Ids of stores which the API needs to be published
+            or updated.
+          schema:
+            type: string
         - $ref: '#/components/parameters/If-Match'
-      requestBody:
-        content:
-          multipart/form-data:
-            schema:
-              properties:
-                file:
-                  type: string
-                  description: WSDL file or archive to upload
-                  format: binary
-                url:
-                  type: string
-                  description: WSDL Definition url
       responses:
         200:
           description: |
             OK.
-            Successful response with updated WSDL definition
+            API was successfully published to all the selected external stores.
           headers:
             ETag:
               description: |
-                Entity Tag of the response resource. Used by caches, or in conditional requests (Will be supported in future).
+                Entity Tag of the blocked subscription.
+                Used by caches, or in conditional requests (Will be supported in future).
               schema:
                 type: string
             Last-Modified:
               description: |
-                Date and time the resource has been modifed the last time.
+                Date and time the subscription has been blocked.
```

Comment:
"was blocked" gives the correct meaning IMO. Please confirm.

Reply:
How about "Date and time of the subscription which was blocked."